frontend/src/global_styles/content/_projects_list.sass

```diff
@@ -134,3 +137,14 @@ body.controller-projects.action-index
     padding-left: 1em
   li
     list-style-type: none
+
+.project-list-container
+  display: flex
+  flex-direction: column
+  max-height: calc(100vh - #{$toolbar-height})
```

Comment:
I think this is not correct any more as you are not calculating based on the toolbar. You define the container like this:
```
$toolbar-height: 80px
max-height: calc(100vh - #{$toolbar-height})
```

But the container is basically a wrapper around **all** content of that page. So what you actually want to substract is the page header and the padding of the `content` which leads to a value of 75px. 
My suggestion would be to extract the padding of the content into a global variable, like we already have for the `header-height`. Thus, you could write here: 

```
max-height: calc(100vh - #{var(--header-height)} - #{var(--content-padding) - #{var(--content-padding)})`
```
 That would at least require no changes if we change the values later on.